src/components/Version.module.scss

```diff
@@ -2,7 +2,7 @@
 
 div.version {
   font-size: 12px;
-  position: absolute;
+  position: fixed;
```

Comment:
this is okay as long as we know it won't overlap any content (how does it look on mobile?)

Reply:
Yes, this overlap content on Mobile. Let me fix.
And just noticed that current prod is also issue on mobile responsive as well on pool page.